### PR TITLE
added ascent exchange tvl

### PR DESCRIPTION
**Pull request by smartcookie0501**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### List of audit links if any:
https://docs.ascent.exchange/security

##### Website Link:
https://ascent.exchange

##### Logo (High resolution, will be shown with rounded borders):
https://i.ibb.co/5815Stm/V4-Cxf32n-400x400.jpg

##### Current TVL: 
v1 - $18,  
v3 - $6.3k

##### Short Description (to be shown on DefiLlama):
The Next-Gen Liquidity Layer on Horizen Eon

##### Token address and ticker if any:
0x01454691Ce0F245D952faD9A6B309A4B5a7D5b0A AEX

##### Github org/user (Optional, if your code is open source, we can track activity):

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/ascentexchange-v3 exports TVL: 
        
 
 
    eon                       6.31 k
    
    total                    6.31 k

**Comment by llamatester** [comment]
The adapter at projects/ascentexchange exports TVL: 
        
 
 
    eon                       18.57829169000015
    
    total                    18.57829169000015

File: package-lock.json
Comment (g1nt0ki) [inline package-lock.json:1]: revert change to this file

File: debug.log
Comment (g1nt0ki) [inline debug.log:1]: remove this file

**Comment by llamatester** [comment]
The adapter at projects/ascentexchange-v3 exports TVL: 
        
 
 
    eon                       6.29 k
    
    total                    6.29 k

**Comment by llamatester** [comment]
The adapter at projects/ascentexchange exports TVL: 
        
 
 
    eon                       18.520660999970143
    
    total                    18.520660999970143

**Comment by g1nt0ki** [comment]
@smartcookie0501 thanks for the PR

**Pull request merged**